File: management-domain/Executive-domain-X/Executive-domain-X/Backend/Java/executive-dashboard-service/Shared/executive-analytics-service/src/main/java/com/gogidix/management/shared/exception/ConflictException.java
package com.gogidix.management.shared.exception;

public class ConflictException extends DomainException {
    public ConflictException(String message) {
        super(message, "CONFLICT");
    }

    public ConflictException(String type, String message) {
        super(type + ": " + message, "CONFLICT");
    }

    public ConflictException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: management-domain/Sales-department/Backend/Node.js/sales-automation-service/src/shared/exceptions/conflict.exception.ts
import { BaseException } from './base.exception';
import { HttpStatus } from '@nestjs/common';

export class ConflictException extends BaseException {
  readonly statusCode = HttpStatus.CONFLICT;
  readonly details: any;

  constructor(message: string, details?: any) {
    super(message);
    this.details = details;
    Object.setPrototypeOf(this, ConflictException.prototype);
  }
}
